### ISSUE TITLE
Windows Defender detects as Trojan:Win32/AgentTesla!ml

### ISSUE THREAD

**Issue by SonicFromSonicTheHedgehog3AndKnuckles** [sonicretro/SonLVL#97]
Have been using this program for a while now and all of a sudden Windows Defender thinks it's malware. Figured I'd report it here.

**Comment by MarioYT21** [comment]
Yeah Windows Defender just thinks it's malware.

**Comment by Brainulator9** [comment]
I have the same issue; my antivirus detects SonLVL as a virus and blocks it.

**Comment by MarioYT21** [comment]
Just turn it off if you wanna keep using SonLVL cause it will just erase it after a little while.

**Comment by Caverns4** [comment]
Yeah, that's a temp fix. I've had Windows occasionally just decide to "forget" to exclude it and automatically delete it anyway once or twice without actually letting me choose to keep it though, so a built-in fix might be helpful. If that's possible.

**Comment by Brainulator9** [comment]
Should be noted: Bitdefender no longer blocks SonLVL.
